.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug Report
+about: Create a report to help us improve image-sequencer
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Please describe the problem (or idea)
+
+> What happened just before the problem occurred? Or what problem could this idea solve?
+
+
+
+> What did you expect to see that you didn't?
+
+
+
+### Please show us where to look
+
+https://beta.sequencer.publiclab.org
```

Comment:
Can we offer some options here too, like https://sequencer.publiclab.org and even localhost?